addons/web/tests/test_assets.py

```diff
@@ -6,15 +6,19 @@
 import odoo
 import odoo.tests
 
+from odoo.tests.common import HttpCase
 from odoo.modules.module import get_manifest
 from odoo.tools import mute_logger
 
+from unittest.mock import patch
+
 _logger = logging.getLogger(__name__)
 
 
 class TestAssetsGenerateTimeCommon(odoo.tests.TransactionCase):
 
     def generate_bundles(self):
+        self.env['ir.attachment'].search([('url', 'like', '/web/assets/%')]).unlink() # delete existing attachement
```

Comment:
IMO, you should use a `=like` instead of a `like` because with `like` the ORM (expression.py) will add the wildcard `%` around `/web/assets/%`